Flickr/Sources/FlickrAppVCs/ErrorCoordinator.swift

```diff
@@ -16,7 +16,13 @@ class ErrorCoordinator {
         case stringLengthInsufficient
     }
 
-    func checkSearchStringForErrors(_ string: String) throws {
+    private enum SearchApiCallErros: Error {
+        case networkError
+        case nilSearchResults
```

Comment:
`nil` is more a swift word. Can you rename it with generic word so that it can be easily understood by other developers too. For ex: `emptySearchResult`

Reply:
Feedback not addressed ?

Reply:
I'm sorry, I had stashed the change due to some confusion and forgot to redo the fix. I have fixed it in the latest commit.